recipe/conda_build_config.yaml

```
@@ -7,5 +7,3 @@ libevent:
   - "2.1.12"
 lmdb:
   - 0.9.22
```

[`aws_sdk_cpp` global pin has been changed a few hours](https://github.com/conda-forge/conda-forge-pinning-feedstock/blame/0a57e320e0f4896490abe0c6f73d43e2b0a9ea83/recipe/conda_build_config.yaml#L268).

This new global pin overrides the version `aws_sdk_cpp` from `1.10.57` (which has been used for `1.6.X`) to `1.11.156` when rerendering https://github.com/conda-forge/arcticdb-feedstock/pull/81/commits/1e05ac23621f72a4f9ebcdd242f286d9a6845242.

Since we do not want to change the previous version, we can override this global configuration by specifying the version here:

```suggestion
  - 0.9.22
aws_sdk_cpp:
  - 1.10.57
```

Then re-rendering will revert the unwanted changes.